psi4/src/psi4/libfock/v.cc

```diff
@@ -1156,6 +1085,10 @@ SharedMatrix RV::compute_gradient() {
     for (size_t i = 0; i < num_threads_; i++) {
         point_workers_[i]->set_deriv(old_deriv);
     }
+    if (functional_->needs_vv10()) {
+        // throw PSIEXCEPTION("V: RKS cannot compute VV10 gradient contribution.");
```

Comment:
i think this throw should be reactivated, until the gradients are implemented in the followup PR

Reply:
Thanks, now throws again.